event.go

```diff
@@ -55,3 +51,45 @@ func createTriggerPayload(channels []string, event string, data interface{}, soc
 		SocketId: socketID,
 	})
 }
+
+func createTriggerBatchPayload(batch []Event, encryptionKey string) ([]byte, error) {
+
```

Comment:
You've left a lot of blank lines in this PR. IMO these are unnecessary (subjective I know!), and not fitting with the existing style.

Reply:
Ok, I'll try to stick to the existing blank line style 👍